cg/server/api.py

```diff
@@ -229,7 +229,7 @@ def samples():
         customer_objs: Optional[Customer] = (
             None if g.current_user.is_admin else g.current_user.customers
         )
-        samples_q: List[Sample] = db.get_samples_by_enquiry(
+        samples_q: List[Sample] = db.get_samples_to_render(
```

Comment:
Not a big fan of this variable name - what does q mean? Queue?

Reply:
Ah sorry, don't review yet, still a draft! AUtomatic PR upload via pycharm makes it request review immediatly, sorry